src/app/fix-for-ngfor/fix-for-ngfor.component.html

```diff
@@ -0,0 +1,15 @@
+<wizard> 
+	<wizard-step title="demo" *ngFor="let wizard of wizardSteps"> 
+		Content of Step 1 
+		Lorem ipsum dolor sic amet,
+		Lorem ipsum dolor sic amet
+		Lorem ipsum dolor sic amet
+		Lorem ipsum dolor sic amet
+		Lorem ipsum dolor sic amet
+		Lorem ipsum dolor sic amet
+		Lorem ipsum dolor sic amet
+		Lorem ipsum dolor sic amet
+		<button type="button" nextStep>Next Step</button> 
```

Comment:
For the buttons I would suggest to use only a next and previous button. Ideally you hide the button if there is no next or previous step.